src/infinity/facets/RetirementQuoter.sol

```diff
@@ -178,17 +172,16 @@ contract RetirementQuoter {
 
     function getUniswapV3Quote(address tokenIn, address tokenOut, uint256 amount)
         internal
-        view
         returns (uint256 additionalSwapAmount)
     {
-        IQuoterView.QuoteExactOutputSingleParams memory params = IQuoterView.QuoteExactOutputSingleParams({
+        IQuoterV2.QuoteExactOutputSingleParams memory params = IQuoterV2.QuoteExactOutputSingleParams({
```

Comment:
QuoterV2 if I recall correctly is meant to be for off-chain only use as it doesn't use view functions and requires a high gas cost on-chain. That's why the IQuoterView is in place here, even though it's not widely documented in Uniswap, so we can get an on-chain quote.

Reply:
Added this file to the repo